app/src/main/java/com/github/raulccabreu/redukt/ui/StateActivity.kt

```diff
@@ -0,0 +1,24 @@
+package com.github.raulccabreu.redukt.ui
+
+import android.support.v7.app.AppCompatActivity
+import com.github.raulccabreu.redukt.Redukt
+import com.github.raulccabreu.redukt.states.StateListener
+
+abstract class StateActivity<T> : AppCompatActivity(), StateListener<T> {
```

Comment:
What do you think about ReduktView, ReduktActivity and ReduktFragment or ReactiveView, ReactiveActivity....